core/src/test/resources/sql/cast-with-format.iq

```diff
@@ -32,8 +32,15 @@ EXPR$0
 2017-05-01 01:23:45
 !ok
 
+# Input that contains shuffled date without time
+select cast('12-2010-05' as timestamp format
+    'DD-YYYY-MM');
+EXPR$0
+2010-05-12 00:00:00
+!ok
+
 !if (false) {
-### disabled until Bug.CALCITE_6269_FIXED ###
+### unsupported format model/elements, test disabled ###
```

Comment:
There is a way in Quidem tests to refer to a Bug, why don't we use that one?
The good thing about the previous version of this string is that you can grep for it, here it is not clear which bug tracks the problem.

Reply:
@mihaibudiu good point, I was also unsure of this comment and ideally wanted to link JIRA issues which would describe the exact causes. But there are quite a number of problems with the current implementation which is causing these many tests to fail and would take time to document them all, what do you recommend be done in this scenario?

Reply:
I think it's best to have an issue to keep track of all of them, even if they are different, and to have an easy way to find it in the code.

The syntax in .iq files is: `!if (fixed.calcite1048)`

Reply:
Updated the comments and disable condition to point to the calcite issue, thanks for suggesting this change!